src/i18n/i18n.ts

```diff
@@ -170,6 +170,8 @@ const dict = {
         branding: "Branding",
         testnet: "Testnet",
         broadcasting_claim: "Broadcasting claim transaction...",
+        open_swap: "Open Swap",
+        swap_in_progress: "This swap is still in progress.",
```

Comment:
Super confusing. You can have swaps in progress even if they are not in local storage. This wording creates too many weird states for the user

Reply:
Looks like this was resolved. I find the current wording understandable.